Frontend/src/components/RankingClass.vue

```diff
@@ -1,24 +1,18 @@
 <template>
```

Comment:
Kann man die Anzahl der Felder auf 3 pro Reihe begrenzen? Die Darstellung ist bei mir mit 6 pro Zeile kaputt. Die Punkte gehen rechts über den Rahmen hinaus.

Ansonsten evtl. die Punkte unterhalb des Namen in darstellen.

Reply:
Ja da ist noch UI Feinarbeit nötig. Die habe ich aber hinten angestellt, da ich davon ausgehe das ich generell für Tabellen noch was anderes verwenden werde. Damit man auch in großen Tabelle suchen kann etc.